cmd/cmd.go

```diff
@@ -344,6 +344,39 @@ func (w *progressWriter) Write(p []byte) (n int, err error) {
 	return len(p), nil
 }
 
+func loadOrUnloadModel(cmd *cobra.Command, opts *runOptions) error {
+	p := progress.NewProgress(os.Stderr)
+	defer p.StopAndClear()
+
+	spinner := progress.NewSpinner("")
```

Comment:
How fast does it stop? Is a spinner needed for feedback?

Reply:
This function is used for both loading/unloading models. In the case of calling stop, it's effectively a NOP for the spinner. It will return immediately and you won't see anything.